### DISCUSSION TITLE
Where do I find Download 1.9,2

### DISCUSSION THREAD

**Discussion by SinisterSticks** [darthcloud/BlueRetro#938] [General]
I was able to find the download on my android however, I can't find the same page on my MacIntosh.  Can someone link to me the download page for version 1.9.2 of the blue retro firmware?

I tried many ways to search on my computer, but I can't find it

**Comment by lamprey13** [comment]
Start here:
https://github.com/darthcloud/BlueRetro/releases

Then Here:
https://darthcloud.itch.io/blueretro
